tests/test_virtual_chassis.py

```diff
@@ -24,3 +27,184 @@ def test_connectivity(self, vct):
                 _, out = dv.runcmd(['sh', "-c", "ping -c 5 -W 0 -q %s" % ip])
                 print(out)
                 assert '5 received' in out
+
+    def test_voq_switch(self, vct):
+        """ Test VOQ switch objects configuration """
+        dvss = vct.dvss
+        for name in dvss.keys():
+            dvs = dvss[name]
+            #Get the config info
+            config_db = DVSDatabase(swsscommon.CONFIG_DB, dvs.redis_sock)
```

Comment:
You should be able to use the `dvs.get_config_db()` method instead here.

Reply:
Yes. Changed as suggested